scripts/e2e-android.sh

```diff
@@ -23,23 +23,24 @@ while [[ $# -gt 0 ]]; do
   shift
 done
 
-# Check if we're running a transaction test
-if [[ $FLOW == *"transaction"* ]]; then
+# Check if we're running all tests or a transaction test
+if [[ $FLOW == *"transaction"* || $FLOW == "e2e" ]]; then
   echo "Transaction test detected."
-  
+
   # Kill any existing Anvil process
   echo "Cleaning up any existing Anvil processes..."
   ANVIL_PID=$(lsof -t -i:8545 -c anvil 2>/dev/null)
   if [ -n "$ANVIL_PID" ]; then
     kill $ANVIL_PID
   fi
   sleep 1
-  
-  # Start Anvil in the background
-  echo "Starting Anvil..."
-  yarn anvil --host 0.0.0.0 &
-  
-  echo "Waiting 5 seconds for Anvil to start..."
+
+  # Start Anvil in the background (show logs in terminal + save to file)
```

Comment:
I moved starting anvil when running all tests here, instead of in the ci script to make it easier to run locally.